ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -14037,20 +14037,11 @@ ABoundActivity * HqlCppTranslator::doBuildActivityDedup(BuildCtx & ctx, IHqlExpr
 
     buildInstancePrefix(instance);
 
+    bool wholeRecord = false;
+    const char *defaultFlag = "";
```

Comment:
I don't think you need this - HDFkeepleft is a fine default for both - and should be the default in the helper classes.